src/zino/tasks/__init__.py

```diff
@@ -8,7 +20,6 @@ def get_registered_tasks():
     from zino.tasks.bfdtask import BFDTask
     from zino.tasks.juniperalarmtask import JuniperAlarmTask
     from zino.tasks.linkstatetask import LinkStateTask
-    from zino.tasks.reachabletask import ReachableTask
     from zino.tasks.vendor import VendorTask
 
-    return [ReachableTask, VendorTask, LinkStateTask, BFDTask, JuniperAlarmTask]
+    return [VendorTask, LinkStateTask, BFDTask, JuniperAlarmTask]
```

Comment:
Why remove ReachableTask from this list? To prevent automated running? Does ReachableTask still need to inherit from task?

Anyway, tasks are crying for a registry/plugin-pattern to be used but that's a question for a different PR.

Reply:
Removed from the list so its not run twice, since its run "manually" once no need to run it again when running the rest.

And yes, with the way this is done inheriting from Task is no longer necessary, which feels kinda bad since it is logically a task... hence why im not sure this is a good solution.

Also yes, a proper plugin thingy is needed. The list was just a quick way to make stuff work initially